vm/src/cairo_run.rs

```diff
@@ -173,7 +173,12 @@ pub fn cairo_run_pie(
         }
     }
     // Load previous execution memory
-    let n_extra_segments = pie.metadata.extra_segments.len();
+    let has_zero_segment = if cairo_runner.vm.segments.has_zero_segment() {
+        0
+    } else {
+        1
+    };
```

Comment:
Just use `as usize`:
```suggestion
    let has_zero_segment = cairo_runner.vm.segments.has_zero_segment() as usize;
```

Reply:
Done